ext/windows/service/daemon.rb

```diff
@@ -54,25 +56,29 @@ def service_main(*argsv)
     #   )
     # end
 
-    puppet = File.join(basedir, 'bin', 'puppet.bat')
+    puppet = File.join(base_dir, 'puppet', 'bin', 'puppet')
+    ruby = File.join(base_dir, 'puppet', 'bin', 'ruby.exe')
+    ruby_puppet_cmd = "\"#{ruby}\" \"#{puppet}\""
```

Comment:
you can interpolate this with `%()` to avoid escaping the double quotes, but I'm fine with this as well:

```ruby
2.6.3 :009 > ruby_puppet_cmd = "\"#{ruby}\" \"#{puppet}\""
 => "\"ruby\" \"puppet\"" 
2.6.3 :010 > ruby_puppet_cmd = %("#{ruby}" "#{puppet}")
 => "\"ruby\" \"puppet\"" 
```